### PR TITLE
Feat/pw-96 career join us

### PR DESCRIPTION
**Pull request by alimarf**
Feature join us is complete

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     "@radix-ui/react-label": "^2.1.0",
     "@radix-ui/react-slot": "^1.1.0",
     "@radix-ui/react-toast": "^1.2.1",
+    "@sanity/block-content-to-react": "^3.0.0",
```
Comment (bimaahida) [inline package.json:20]: where do we use that library?

File: package.json [hunk 2]
```diff
@@ -31,6 +32,7 @@
     "react-dom": "^18",
     "react-google-recaptcha": "^3.1.0",
     "react-hook-form": "^7.53.0",
+    "react-refractor": "^3.1.1",
```
Comment (bimaahida) [inline package.json:35]: i do not see where we use this also

File: app/careers/join-us/page.tsx [hunk 3]
```diff
@@ -30,67 +39,63 @@ export default function JoinUs() {
...
-
-          <p className="text-sm text-primary font-medium mb-2">
-            Or Email:
+          <p className="text-[15px] lg:text-lg text-primary font-medium mb-2">
+            Or Email :
           </p>
 
-          <a href="mailto:recruitment@arthawena.com" className="text-xs lg:text-sm font-medium">
-            recruitment@arthawena.com
+          <a href="mailto:recruitment@arthawena.com" className="text-[15px] lg:text-lg font-medium">
```
Comment (bimaahida) [inline app/careers/join-us/page.tsx:84]: for href please following the cms value

File: package.json (hunk 1 above)
Comment (alimarf) [reply package.json:20]: forget to remove this one, xixixi

File: package.json (hunk 2 above)
Comment (alimarf) [reply package.json:35]: need to remove this one as well, xixixix

File: app/careers/join-us/page.tsx (hunk 3 above)
Comment (alimarf) [reply app/careers/join-us/page.tsx:84]: okey Mr. eagle eye

**Review by bimaahida — APPROVED** [review]
LGTM 💸

**Pull request merged**